css/index.css

```diff
@@ -121,32 +123,65 @@ header .logo {
  
 }
 
-img{
-  width: 50px;
-  height:auto;
-
-}
-
 
 /* ticker */
 .ticker {
   color: rgb(159, 77, 137);
   font-family: Lucida Handwriting;
   font-size: 1.25rem;
-  width: 70%;
+  width: 65%;
   height: 10%; 
   padding: 10px;
-  margin-bottom: 20px;
-  margin-left: 200px; 
+  margin-bottom: 5px;
+  margin-left: 260px; 
+
+}
+
+.grid-container{
+  display: grid;
+  grid-template-columns: 1fr 3.5fr 1fr;
 }
 
+/* animation */
+
+.container-a {
+  position: relative;
+   width: 1fr;  
+   height: 100%; 
+   margin: 1em;
+   overflow: hidden; 
+   pointer-events: none;
+ }
+ 
+ .flower::before {
+   content: "";
+   position: absolute;
+   background: url(cosmos-transformed.png);
+   width: 100%;
+   height: 100%;
+   background-repeat: no-repeat;
+   background-size: contain;
+   background-position: center;
+   animation: move 5s linear infinite ;
+ } 
+ 
+ @keyframes move {
+   0% {
+     transform: translate(0) rotate(0deg);
+   }
+   100% {
+     transform: translate(300px) rotate(360deg);
+   }
+ }
+   
```

Comment:
This animation is awesome. It really adds a personal touch to the page. Even the flower image works with the colors you've chosen for the page.